addons/website/tests/test_performance.py

```diff
@@ -3,16 +3,13 @@
 
 from odoo.tests.common import HttpCase
 
-EXTRA_REQUEST = 4 - 1
+EXTRA_REQUEST = 2 - 1
 """ During tests, the query on 'base_registry_signaling, base_cache_signaling'
-won't be executed on hot state, but new queries related to the test cursor
-will be added::
+won't be executed on hot state, but new queries related to the test cursor will
+be added:
```

Comment:
This just creates unnecessary diff-churn.

Reply:
Indeed, thank for mentioning.
Since I am editing that comment and the constant value, I wanted to fix that `::` typo (that actually comes from https://github.com/odoo/odoo/commit/17e6a69b9189127f2988d05850d2594244c2ff2c where the un-indent also seems like an useless diff in the first place AFAIK, even if I do agree it is better written without the indent level)

Since I am fixing that `::` part, I might as well rewrite both sentences to make them use the available 80 char space while I am touching it.